mantis-common/src/main/java/io/mantisrx/common/metrics/measurement/MicrometerMeasurement.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2023 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.mantisrx.common.metrics.measurement;
+
+import io.micrometer.core.instrument.Meter;
+
+public class MicrometerMeasurement {
```

Comment:
Do we even need this type, or can we convert the micrometer measurements to the existing measures DS?

Reply:
I think we should keep them separate to allow identify mantis metrics vs micrometer metrics. 
Eventually, we can migrate Measurements to MicrometerMeasurement only.